### PR TITLE
DNS_CLUSTER_IP cannot be specified in IPv6 Cluster

### PR DESCRIPTION
**Pull request by literalice**
*Description of changes:*

`DNS_CLUSTER_IP` isn't recognized in a IPv6 cluster. It is fixed to the IP from `SERVICE_IPV6_CIDR`.

The PR enables setting `DNS_CLUSTER_IP` or `--dns-cluster-ip` in IPv6 env.

### CODE REVIEW

File: files/bootstrap.sh
Comment (literalice) [inline files/bootstrap.sh:389]: It's run only when `--dns-cluster-ip` isn't specified in this PR.

File: files/bootstrap.sh
Comment (ravisinha0506) [inline files/bootstrap.sh:388]: Can we update this error message saying "either service ipv6 cidr or cluster-dns-ip need to be present"?

Comment (bwagner5) [inline files/bootstrap.sh:388]:
```suggestion
      echo "Either --service-ipv6-cidr or --cluster-dns-ip must be provided when --ip-family is set to ipv6"
```

**Review by bwagner5 — APPROVED** [review]
If the new error message is good with @ravisinha0506 then, lgtm

**Review by suket22 — APPROVED** [review]
Changes LGTM. 

I suppose this could only break those that are currently setting **both**  `--dns-cluster-ip` && `--service-ipv6-cidr`, and have been setting an incorrect value for the former. Everything is working fine for them right now because the latter is overriding the former. I don't expect anybody's currently doing this though so let's move forward. 


For the future - I think we can further reduce the complexity in the bootstrap script by never accepting `--ip-family` at all.

Please wait for @ravisinha0506 for the final approval and merge.

**Pull request merged**
